adminguide.rst

```diff
@@ -498,6 +498,8 @@ TOML table, as shown in the example below:
 
 .. code-block:: toml
 
+  # /etc/flux/system/conf.d/ingest.toml
```

Comment:
This seems fine, but I was wondering if our implication that the toml config for every subsystem has to be in a different file is going to drive sysadmins crazy. Is there some benefit to implying that?

Reply:
Yeah good point.   I was just going for consistency here, but I'm open to changing the overall presentation.

Reply:
Yeah, I was more thinking out loud at this point